go.mod

```diff
@@ -81,4 +83,6 @@ require (
 	nhooyr.io/websocket v1.8.6 // indirect
 )
 
+replace github.com/ipfs/go-unixfs => github.com/ipfs/go-unixfs v0.2.2
```

Comment:
Why this instead of setting the version directly?

Reply:
Becase a lot of libs in this ecosystem use v0. That means that the dependency graph is very brittle when you include a new dependency that ends up using a newer version of an indirect dependency considering how Go MVS version resolving works.

A lot of libs in the IPFS ecosystem that are v0 make breaking changes in further v0.. and that doesn't play well with Go MVS (which selects the maximum referenced minor version of a major version). 

Yes... horrible. Usually you have some luck by trying to upgrade other deps hoping that they will stop reference the old v0 version that is causing this needed downgrade, but that's not usually an easy task.

Reply:
Long story short, if you don't use strict semantic versioning in the Go ecosystem.. you'll create nightmares for your clients.

Reply:
Okay I see this is to force its direct and indirect dependencies to use this specific version instead of relying on MVS. Maybe worth a comment? Or maybe I'm ignorant enough to not understanding the intention in the first place ;)

Reply:
Yes, that's exactly the intention. We usually only use `replace` to solve MVS problems on dependencies that don't follow strict semantic versioning. So that's the default correct interpretation.